pallets/gear/src/benchmarking/mod.rs

```diff
@@ -1472,6 +1485,18 @@ benchmarks! {
         verify_process(res.unwrap());
     }
 
+    mem_grow {
+        let r in 0 .. API_BENCHMARK_BATCHES;
+
+        let mut res = None;
+        let exec = Benches::<T>::mem_grow(r, 1)?;
```

Comment:
I don't see any reason to calculate grow as syscall call that contain interruption just because usually it's called by alloc already at backend side. What do you think, @grishasobol ?